script/Init.s.sol

```diff
@@ -82,7 +83,8 @@ contract Init is BaseScript {
                 asset: asset,
                 cancelable: true,
                 transferable: true,
-                broker: Broker(address(0), ud60x18(0))
+                broker: Broker(address(0), ud60x18(0)),
+                shape: "Exponential Dynamic"
```

Comment:
Yes. It would.

Reply:
This is an `Init` script which is used to launch dummy streams when the lockup protocol is launched on a new chain. We can definitely refactor it to include a more popular shape, however, the time it would take to refactor (even small) would not justify the utility of this script.